python/looker_sdk/sdk/api40/methods.py

```diff
@@ -4675,17 +4675,17 @@ def query_task(
     # Query Tasks whose results have expired will have a status of 'expired'.
     # If the user making the API request does not have sufficient privileges to view a Query Task result, the result will have a status of 'missing'
     #
-    # GET /query_tasks/multi_results -> MutableMapping[str, str]
+    # GET /query_tasks/multi_results -> MutableMapping[str, Any]
```

Comment:
it still feels like we're over-reaching - in this case I think the results are actually meant to be `str, str`?

Reply:
The problem right now is the spec indicates `str, str` for hash types incorrectly for more than `vis_config`. Creating the "fix this" list is as much work as actually fixing it, so we lose *some* strong typing by downgrading to `Any` but most importantly things can be deserialized and also serialized back to what our endpoints require.

Reply:
sounds right.